src/oracles/UniV3Oracle.sol

```diff
@@ -51,6 +61,7 @@ contract UniV3Oracle is INFTOracle {
         view
         returns (
             bool success,
+            bool deviationSafety,
```

Comment:
Why don't we use a single success flag for all cases where the returned price is not accurate?